lms/djangoapps/verify_student/views.py

```diff
@@ -254,11 +258,13 @@ def get(  # lint-amnesty, pylint: disable=too-many-statements
         # 2) If set, verification deadlines are always AFTER upgrade deadlines, because why would you
         #   let someone upgrade into a verified track if they can't complete verification?
         #
-        verification_deadline = VerificationDeadline.deadline_for_course(course.id)
-        response = self._response_if_deadline_passed(course, self.VERIFICATION_DEADLINE, verification_deadline)
-        if response is not None:
-            log.info("Verification deadline for '%s' has passed.", course.id)
-            return response
+        verification_deadline = None
+        if not is_integrity_signature_enabled(course_key):
```

Comment:
#NIT Can you add an else statement and log that the verification is bypassed because the flag is on?